src/lib/y2security/selinux.rb

```diff
@@ -145,21 +220,39 @@ def configurable?
     GETENFORCE_PATH = "/usr/sbin/getenforce".freeze
     private_constant :GETENFORCE_PATH
 
-    # Find SELinux mode by given value
+    # Returns the values for the SELinux setting from the product features
     #
-    # @note using nil means to set SELinux mode as disabled.
+    # @return [Hash{Symbol => String, Boolean, nil}] e.g., { mode: "enforcing", configurable: false }
+    #   a hash holding the defined SELinux options in the control file;
+    #   an empty object if no settings are defined
+    def product_feature_settings
+      @product_feature_settings unless @product_feature_settings.nil?
+
+      settings = Yast::ProductFeatures.GetFeature("globals", "selinux")
+      settings = {} if settings.empty?
```

Comment:
so it can be empty string? I do not get this line

Reply:
If I'm not wrong, when it is not defined ProductFeatures returns and empty string. See https://github.com/yast/yast-yast2/blob/e914214656d217ae05a51f6347e22b520caae907/library/control/src/modules/ProductFeatures.rb#L203-L213

Reply:
yep, as default is not there, it is empty string, so fine for me.